src/test/kotlin/day03/ClericTest.kt

```
@@ -0,0 +1,48 @@
+package day03
+
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+import org.junit.Assert.*
+ class ClericTest {
+
+@Before
+ fun setUp() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)